src/oauth2/controller/authorize.ts

```diff
@@ -108,16 +117,29 @@ class AuthorizeController extends Controller {
       codeChallenge: params.codeChallenge,
       codeChallengeMethod: params.codeChallengeMethod,
       browserSessionId: ctx.sessionId!,
+      nonce: params.nonce,
     });
 
+    const redirectParams: Record<string, string> = {
+      code: code.code,
+    };
+    if (params.state) redirectParams.state = params.state;
+
+    if (params.responseType === 'code id_token') {
+      redirectParams.id_token = await generateJWTIDToken({
```

Comment:
Not an expert, but I thought the ID token is meant to be included in the token response. Just making sure this isn't an oversight